gui/src/main/java/org/jboss/as/console/client/shared/runtime/ws/WebServiceRuntimeView.java

```diff
@@ -215,5 +215,14 @@ public void updateEndpoints(List<WebServiceEndpoint> endpoints) {
         ColumnSortEvent.fire(table, table.getColumnSortList());
 
         table.selectDefaultEntity();
+        SingleSelectionModel<WebServiceEndpoint> selectionModel =
+                (SingleSelectionModel<WebServiceEndpoint>) table.getSelectionModel();
+        WebServiceEndpoint selection = selectionModel.getSelectedObject();
```

Comment:
Let's add a private method 

```java
private void addSample(WebServiceEndpoint endpoint) {
    if endpoint != null) {
        sampler.addSample(new Metric(
            selection.getRequestCount(),
            selection.getResponseCount(),
            selection.getFaultCount()));
    }
}
```

which is called from the selection changed handler and the `updateEndpoints` method.